packages/ember-glimmer/lib/renderer.js

```diff
@@ -132,7 +132,7 @@ class Renderer {
     let self = new RootReference(view);
     let dynamicScope = new DynamicScope({
       view,
-      controller: view.outletState.render.controller,
+      controller: view.outletState.outlets.main.render.controller,
```

Comment:
This seems correct based on this? https://github.com/emberjs/ember.js/blob/master/packages/ember-htmlbars/lib/keywords/outlet.js#L95